awxkit/awxkit/api/pages/api.py

```diff
@@ -62,6 +62,7 @@
     ('Group', 'potential_children'),
     ('Host', 'all_groups'),
     ('WorkflowJobTemplateNode', 'create_approval_template'),
+    ('Credential', 'CredentialInputSource'),
```

Comment:
This is definitely incorrect.  It won't do anything at all, since the second parameter in each of these in this list is supposed to be the name of a related endpoint.

Reply:
not to mention that the point of this list is to *stop* things from being exported.